index.bs

```diff
@@ -1,317 +1,215 @@
 <pre class='metadata'>
-Title: CSS Display Locking
-Shortname: css-display-locking
+Title: CSS Render Subtree
+Shortname: css-render-subtree
 Level: 1
 Status: UD
 Group: csswg
 Work Status: exploring
-URL: https://tabatkins.github.io/specs/css-display-locking
+URL: https://wicg.github.io/display-locking
 Editor: Tab Atkins-Bittner, Google
-Abstract: This spec introduces features for "display locking",
-	allowing authors to more easily control when rendering work occurs,
-	making large amounts of content more performant.
+Editor: Vladimir Levin, Google
+Abstract: This spec introduces features for "display locking", allowing authors
+          to provide rendering performance hints to the user-agent. The user-agent can
```

Comment:
They aren't really hints. knobs instead?

Reply:
I really don't know what to call it... I don't think they are knobs.. I think they are "very strong hints" Is there a better word than either hints or knobs?